README.md

```diff
@@ -71,12 +71,34 @@ npm install --save gridstack
 
 ## Include
 
-after you install:
+* Using webpack, you can now import the files:
 
 ```js
-import 'gridstack/dist/gridstack.all.js';
+import 'jquery';
+import 'jquery-ui';
+
+import 'gridstack/dist/gridstack-poly'; // optional IE support
+import GridStack from 'gridstack/dist/gridstack';
+import 'gridstack/dist/gridstack.jQueryUI';
 import 'gridstack/dist/gridstack.css';
 ```
+
+If your project doesn't make use of jQuery or jQueryUI and you don't want to include these libs,
```

Comment:
actually they can still use jquery in their code, it's the official latest one I include. Only thing is jquery-ui is a trimmed down version (1/5th the size) that has just what you need for drag&drop, so if you don't need all jquery-ui yourself you might want to include this smaller one. You can bring whichever jquery (1.8.3+ I believe) you want.

Reply:
ok so it may be better for webpack to recommend jquery 1.8.3+ managed through npm and just mention the smaller jQueryUI, is that ok for you ?

Reply:
well I don't want to expose jquery dependencies as many might not use this lib otherwise, which is why we don't have an external dependency and all.js include everything you need to run. JQ was removed from 1.x API signature, and from the main code in 2.x, and eventually out and replaced with html5 drag&drop.

Reply:
Also is it not possible to have
```js
import 'gridstack/dist/gridstack.all.js'
```
anymore ? I thought that worked... and hides the internal usage of jq

Reply:
Also what about the jquery.js change I also had made
`
// Expose jQuery and $ identifiers, even in AMD
// (#7102#comment:10, https://github.com/jquery/jquery/pull/557)
// and CommonJS for browser emulators (#13566)
//if ( !noGlobal ) { [alain] noGlobal=true when using webpack/TS, yet we compile this in, not AMD load, so always expose
	window.jQuery = window.$ = jQuery;
//}
`